socorro/processor/processor_app.py

```diff
@@ -165,7 +165,15 @@ class ProcessorApp(App):
             "a local filesystem path that can be used as a workspace for processing "
             + "rules"
         ),
-        default=os.path.join(tempfile.gettempdir(), "workspace"),
+        default=os.path.join(
+            (
+                # FIXME(willkg): this is for backwards compatability with existing
+                # configuration
+                os.environ.get("resource.boto.temporary_file_system_storage_path", "")
```

Comment:
If this environment variable exists, it takes this value as the default value. If not, it takes `tempfile.gettempdir()` (which is a more sensible default).

This will bridge the current state of main with the current configuration in stage/prod until we update stage/prod configuration.